src/Microsoft.AspNet.Mvc.Core/CacheProfile.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.Mvc
+{
+	public class CacheProfile
+	{
+        /// <summary>
+        /// Gets or sets the name of the CacheProfile.
+        /// The CacheProfile is referred in the ResponseCache attribute using this name.
+        /// </summary>
+        public string Name { get; set; }
+
+		/// <summary>
+        /// Gets or sets the duration in seconds for which the response is cached.
+        /// This is a required parameter.
+        /// This sets "max-age" in "Cache-control" header.
+        /// </summary>
+        public int? Duration { get; set; }
```

Comment:
You can not mention the duration in the CacheProfile and you can override this in the attribute.